appserver/admingui/healthcheck-service-console-plugin/src/main/resources/fish/payara/admingui/healthcheck/Strings.properties

```diff
@@ -70,6 +70,8 @@ healthcheck.configuration.stuckThreadsCheckerTabTitle=Stuck Threads
 healthcheck.configuration.stuckThreadsCheckerTabTitleToolTip=Configures the Stuck Threads HealthCheck.
 healthcheck.configuration.garbageCollectorCheckerTabTitle=Garbage Collector
 healthcheck.configuration.garbageCollectorCheckerTabTitleToolTip=Configures the Garbage Collector HealthCheck.
+healthcheck.configuration.mircroprofileHealthcheckCheckerTabTitle=Microprofile Healthcheck
```

Comment:
To avoid confusion with MicroProfile Healthcheck. I think the title should have "checker" at the end.